File: Question1.c
#include <stdio.h>
int MultiplyFactor(int iNo)
{
    if(iNo <0)
    {
        iNo = -iNo ;
    }

    int iCnt= 0;
    int iFactor =0;
    iFactor = 1 ;
    for (iCnt =1 ;iCnt <= (iNo /2); iCnt ++)
    {
        if (iNo % iCnt == 0  )
        {   
            iFactor = iFactor * iCnt;
        }
    }
    return iFactor;

}

int main()
{
    int iValue = 0 ;
    int iRet = 0;
    printf("Enter Number : ");
    scanf("%d",&iValue);
    iRet = MultiplyFactor(iValue);

    printf("%d is the Multiplication of Factor",iRet);
   
   

    return 0 ;
}

File: Assignment 20/Question1.java
import java.util.Scanner;

class Logic
{
    
    public int PrintEven(int iNo) 

    {
        int iCnt = 0;
        int iSum = 0;
        for (iCnt = 2 ; iCnt<=iNo ; iCnt= iCnt+2)
        {
            iSum = iSum + iCnt ;
               
        } 
        return iSum; 

       
    }  
}          
class Question1
{
    public static void main(String[] args) 
    {
        
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0 , iRet = 0 ;
       
        System.out.println("Enter your Number :");
        iValue1 = sobj.nextInt();

        Logic lobj = new Logic();

        iRet = lobj.PrintEven(iValue1);
        System.out.println("The sum is " +iRet);  
    }    
}
